resources/views/components/editor.blade.php

```diff
@@ -13,48 +13,50 @@
+                <x-button flat squared x-on:click="editor().chain().focus().toggleStrike().run()" class="line-through" label="S"></x-button>
             @endif
             @if($code)
-                <x-button flat squared x-on:click="editor().chain().focus().toggleCode().run()" icon="code"/>
+                <x-button flat squared x-on:click="editor().chain().focus().toggleCode().run()" icon="code" label=" "/>
             @endif
             @if($h1)
-                <x-button flat squared x-on:click="editor().chain().focus().toggleHeading({ level: 1 }).run()">H1</x-button>
+                <x-button flat squared x-on:click="editor().chain().focus().toggleHeading({ level: 1 }).run()" label="H1"></x-button>
             @endif
             @if($h2)
-                <x-button flat squared x-on:click="editor().chain().focus().toggleHeading({ level: 2 }).run()">H2</x-button>
+                <x-button flat squared x-on:click="editor().chain().focus().toggleHeading({ level: 2 }).run()" label="H2"></x-button>
             @endif
             @if($h3)
-                <x-button flat squared x-on:click="editor().chain().focus().toggleHeading({ level: 3 }).run()">H3</x-button>
+                <x-button flat squared x-on:click="editor().chain().focus().toggleHeading({ level: 3 }).run()" label="H3"></x-button>
             @endif
 
             @if($horizontalRule)
-                <x-button flat squared x-on:click="editor().chain().focus().setHorizontalRule().run()">-</x-button>
+                <x-button flat squared x-on:click="editor().chain().focus().setHorizontalRule().run()" label="-"></x-button>
             @endif
 
             @if($bulletList)
                 <x-button flat squared x-on:click="editor().chain().focus().toggleBulletList().run()">
+                    <x-slot:label>
                         <x-heroicons name="list-bullet" class="w-4 h-4"/>
+                    </x-slot:label>
                 </x-button>
             @endif
             @if($orderedList)
                 <x-button flat squared x-on:click="editor().chain().focus().toggleOrderedList().run()">
+                    <x-slot:label>
                         <x-heroicons name="list-bullet" class="w-4 h-4"/>
+                    </x-slot:label>
                 </x-button>
             @endif
             @if($quote)
-                <x-button flat squared x-on:click="editor().chain().focus().toggleBlockquote().run()">
-                    „“
-                </x-button>
+                <x-button flat squared x-on:click="editor().chain().focus().toggleBlockquote().run()" label="„“" />
             @endif
             @if($codeBlock)
-                <x-button flat squared x-on:click="editor().chain().focus().toggleCodeBlock().run()">
+                <x-button flat squared x-on:click="editor().chain().focus().toggleCodeBlock().run()" label=" ">
```

Comment:
set to null on code and used slot:label instead of slot on codeBlock.
label is set in the editor component and inherited by the button components, so all components that dont have a label set get the label provided to `<x-editor label="abc" />`